lib/zendesk_apps_support/package.rb

```diff
@@ -190,7 +190,7 @@ def has_location?
     deprecate :has_location?, 'manifest.location?', 2016, 9
 
     def has_file?(path)
-      File.exist?(path_to(path))
+      File.file?(path_to(path))
```

Comment:
did we ever use this to check if there's an assets folder or anything like that?